content/en/logs/explorer/search_syntax.md

```diff
@@ -113,13 +115,43 @@ Wildcard searches work within tags and attributes (faceted or not) with this syn
 service:*mongo
 ```
 
-Wildcard searches can also be used to search in the plain text of a log that is not part of a facet. This query returns all the logs that contain the string `NETWORK`:
+Wildcard searches can also be used to search in the plain text of a log that is not part of a log attribute. For example, this query returns all the logs that contain the string `NETWORK`:
 
 ```
 *NETWORK*
 ```
 
-However, this search term does not return logs that contain the string `NETWORK` if it is in a facet and not part of the log message.
+However, this search term does not return logs that contain the string `NETWORK` if it is in a log attribute and not part of the log message.
+
+### Full-text search across all log attributes
+
+<div class="alert alert-warning">The full-text search feature is in beta. It is only available in Log Management and works in monitor, dashboard, and notebook queries. The full-text search syntax cannot be used for defining index filters, archive filters, log pipeline filters, or in Live Tail. </div>
+
+<div class="alert alert-warning">You can use full-text search for all logs ingested after MONTH DATE YEAR.</div>
+
+Use the syntax `*:search_term` to perform a free text search across all log attributes, including the log message.
+
+#### Single term example
```

Comment:
Maybe let's add an example of the difference between `*:kelner` vs `*:*kelner*` ? See these two screenshots

![Screenshot 2024-03-26 at 2 40 07 PM](https://github.com/DataDog/documentation/assets/133862/f55df332-b7fe-45d5-ba9d-de490e0968f6)
vs
![Screenshot 2024-03-26 at 2 40 29 PM](https://github.com/DataDog/documentation/assets/133862/18fe9b53-0e11-45f1-b838-a2f270736087)

1,308 results vs 1,907,477 results

Reply:
@ckelner Added an example below the single term example, is that what you are looking for?

Reply:
DM'd this works 🎉